config/locales/flood_risk_engine/enrollments/steps/show/generic.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
The generic locale file was dropped in a previous PR - is there a reason why you are bringing it back - or is this an unwanted merge relic?